Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ControlSchemesView.cs

```diff
@@ -57,7 +57,7 @@ public ControlSchemesView(VisualElement root, StateContainer stateContainer, boo
             var popupWindow = new PopupWindow
             {
                 text = "Add Control Scheme",
-                style = { position = new StyleEnum<Position>(Position.Absolute) }
+                style = { position = new StyleEnum<Position>(Position.Absolute), maxWidth = new StyleLength(new Length(100, LengthUnit.Percent)) }
```

Comment:
Saw this already got merged, but I wonder if the right thing to do here was to constrain the window, wasn't it the text input field that was supposed to be constrained in size?